CONTRIBUTING.md

```diff
@@ -2,35 +2,36 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Contributor License Agreements](#contributor-license-agreements)
-- [Finding Things That Need Help](#finding-things-that-need-help)
-- [Versioning](#versioning)
-  - [Codebase and Go Modules](#codebase-and-go-modules)
-    - [Backporting a patch](#backporting-a-patch)
-  - [APIs](#apis)
-  - [CLIs](#clis)
-- [Branches](#branches)
-  - [Support and guarantees](#support-and-guarantees)
-  - [Removal of v1alpha3 & v1alpha4 apiVersions](#removal-of-v1alpha3--v1alpha4-apiversions)
-- [Contributing a Patch](#contributing-a-patch)
-- [Documentation changes](#documentation-changes)
-- [Releases](#releases)
-- [Proposal process (CAEP)](#proposal-process-caep)
-- [Triaging E2E test failures](#triaging-e2e-test-failures)
-- [Reviewing a Patch](#reviewing-a-patch)
-- [Reviews](#reviews)
-  - [Approvals](#approvals)
-- [Features and bugs](#features-and-bugs)
-- [Experiments](#experiments)
-- [Breaking Changes](#breaking-changes)
-- [API conventions](#api-conventions)
-  - [Optional vs. Required](#optional-vs-required)
-    - [Example](#example)
-    - [Exceptions](#exceptions)
-  - [CRD additionalPrinterColumns](#crd-additionalprintercolumns)
-- [Google Doc Viewing Permissions](#google-doc-viewing-permissions)
-- [Issue and Pull Request Management](#issue-and-pull-request-management)
-- [Contributors Ladder](#contributors-ladder)
+- [Contributing Guidelines](#contributing-guidelines)
```

Comment:
Please rebase to main and run `make generate-doctoc`

Reply:
Although this will just revert your change. If we want to change the toc. Let's please fix the headlines and then regenerate and not edit manually